api/src/main/java/org/openmrs/module/fhir2/api/util/LastnResults.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.util;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class LastnResults {
+	
+	String uuid;
+	
+	Date datetime;
+	
+	Map<String, Object> attributes;
+	
+	public LastnResults(Object uuid, Object datetime, Map<String, Object> attributes) {
+		this.uuid = (String) uuid;
+		this.datetime = (Date) datetime;
+		this.attributes = attributes;
+	}
+	
+	public LastnResults(Object[] objects) {
+		uuid = (String) objects[0];
+		datetime = (Date) objects[1];
+	}
+	
+	public Map<String, Object> getAttributes() {
+		return attributes;
+	}
+	
+	public String getUuid() {
+		return uuid;
+	}
+	
+	public void setUuid(String uuid) {
+		this.uuid = uuid;
+	}
+	
+	public Date getDatetime() {
+		return datetime;
+	}
+	
+	public void setDatetime(Date datetime) {
+		this.datetime = datetime;
+	}
+	
+	public void setAttributes(Map<String, Object> attributes) {
+		this.attributes = attributes;
+	}
+	
+	public static List<String> getTopNRankedUuids(List<LastnResults> list, int max) {
```

Comment:
It's not a good idea to have this `LastnResult` model inside the utils class. It would be better to define the model in the dao.impl package whereas this method can reside in a separate utils class (public final) called maybe `LastnOperationUtils.java`.

Reply:
Can we have a package called `model` in `dao` package for `LastnResult` ?

Reply:
I'm actually in favour of having this in the `util` folder. `model` is primarily intended for the Hibernate objects that we've needed to add over time (`GroupMember` being a bit of an oddity). I don't think we need a sub-package for dao...